vue.config.js

```diff
@@ -1,8 +1,28 @@
 module.exports = {
+	parallel: process.env.NODE_ENV !== 'production',
```

Comment:
can you add a comment why we disable it?

Reply:
added :)